packages/client/hmi-client/src/components/drilldown/tera-output-dropdown.vue

```diff
@@ -31,14 +39,29 @@
 <script setup lang="ts">
 import { WorkflowOutput, WorkflowPortStatus } from '@/types/workflow';
 import Dropdown from 'primevue/dropdown';
+import { getElapsedTimeText } from '@/utils/date';
 
-defineProps<{
+const props = defineProps<{
 	options: WorkflowOutput<any>[] | { label: string; items: WorkflowOutput<any>[] }[];
 	output: WorkflowOutput<any>['id'];
 	isLoading?: boolean;
 }>();
 
 const emit = defineEmits(['update:selection']);
+
+const getOptionById = (id, items) => items.find((option) => option.id === id);
+
+const getCreateTimeById = (id: string) => {
+	const items = props.options.flatMap((option) => option.items);
+	const option = getOptionById(id, items);
+	return getElapsedTimeText(option.timestamp);
+};
+
+const getLabelById = (id: string) => {
+	const items = props.options.flatMap((option) => option.items);
```

Comment:
This is repeated in `getCreateTimeById`, maybe `items` should be a `computed` variable

Reply:
changed to computed